python/grass/script/core.py

```diff
@@ -740,7 +740,8 @@ def debug(msg, debug=1):
     """Display a debugging message using `g.message -d`
 
     :param str msg: debugging message to be displayed
-    :param str debug: debug level (0-5)
+    :param str debug: debug level (0-5) which is controlled by
+                      the ``DEBUG`` GRASS gisenv variable (set with `g.gisenv`)
```

Comment:
This seems misleading. Why it is set a parameter when it is controlled by DEBUG?

Reply:
Yes, right, the `g.gisenv` debug messages are unrelated to this. I'll fix this PR.